README.md

```diff
@@ -34,6 +34,8 @@ include ::profile_lustre::lnet_router
 
 ## Usage
 
+WARNING: v3 of this module introduces changes to names and structuring of some parameters.
```

Comment:
I think we also need to call out that if upgrading between versions, users will need to `rm /etc/lnet.trigger` to trigger Puppet to reconfigure the LNet with the new settings.

And/or perhaps the module should trigger a reconfigure of LNet if `/etc/lnet.conf` and/or `/etc/modprobe.d/lustre.conf` are updated (I'm not clear which files being updated should actually trigger this).

Reply:
I updated to reference the more specific v3.0.0, and updated README.md quite a bit more.

I overlooked your comment about the trigger file so let me write something about that as well.

A reconfig of LNet would require unmounting Lustre and might cause problems (or least probably would fail) if it were attempted with active mounts, so I'm in favor of leaving the trigger.

Reply:
Or if when it fails with the trigger in place, could the output of that Puppet run somehow suggest that you may need to trigger a rebuild?

Reply:
I've added a note about the trigger file in README.md in a 3rd commit.

Reply:
I opened a ticket for the idea of having Puppet notify when config files have been updated but LNet hasn't been reconfigured: https://jira.ncsa.illinois.edu/browse/SVCPLAN-5245

It's a good idea but I want to think about it a bit more, and I'd like to get this branch merged soonish, if possible.

Reply:
I still think mentioning that the LNet would need to be reconfigured in the `WARNING: v3.0.0 ...` section would be useful. That was not obvious to me when I tested this branch.

Reply:
Got it, I didn't compute that you were wanting it in that particular place. I've added it in a 4th commit.